src/resolvers/Mutations.js

```diff
@@ -292,6 +292,112 @@ async function deleteTeam(_, args, context){
     throw new UserInputError("Team does not exist");
 }
 
+async function createApproval(_, args, context, info){
+
+    var address = getNewAddressFromArgs(args.requestedChange);
+    
+    return await context.prisma.mutation.createApproval({
+        data: {
+            gcIDApprover: {connect: {gcID: copyValueToObjectIfDefined(args.gcIDApprover.gcID)}},
+            gcIDSubmitter: {connect: {gcID: copyValueToObjectIfDefined(args.gcIDSubmitter.gcID)}},
+            requestedChange: {
+                create:{
+                    gcID: args.requestedChange.gcID,
+                    name: args.requestedChange.name,
+                    email: args.requestedChange.email,
+                    avatar: args.requestedChange.avatar,   
+                    mobilePhone: args.requestedChange.mobilePhone,
+                    officePhone: args.requestedChange.officePhone,
+                    address: {create:address},
+                    titleEn: args.requestedChange.titleEn,  
+                    titleFr: args.requestedChange.titleFr,
+                    team:{
+                        create:{
+                            nameEn:args.requestedChange.team.nameEn,
+                            nameFr:args.requestedChange.team.nameFr,
+                            descriptionEn:args.requestedChange.team.descriptionEn,
+                            descriptionFr:args.requestedChange.team.descriptionFr,
+                            colour:args.requestedChange.team.colour, 
+                            avatar:args.requestedChange.team.avatar,
+                            organization:{connect:{ id:args.requestedChange.team.organization.id}},
+                            owner:args.requestedChange.team.owner,                        
+                        }
+                    },       
+                }
+            },
+            createdOn: args.createdOn,
+            actionedOn: copyValueToObjectIfDefined(args.actionedOn),
+            deniedComment: copyValueToObjectIfDefined(args.deniedComment),
+            status: args.status,
+            changeType: args.changeType
+        }
+    }, info);
+}
+
+async function modifyApproval(_, args, context, info){
+    var address = getNewAddressFromArgs(args.data.requestedChange);
```

Comment:
Can remove any reference to be able to modify Requested changes.  The only fields available to change should be status and deniedComment.